### ISSUE TITLE
feature request: PBKDF2 Support for User Import

### ISSUE THREAD

**Issue by ColaIan** [logto-io/logto#6870]
### What problem did you meet?

https://docs.logto.io/user-management/user-migration#password-hashing
I'm trying to migrate from authentik to logto, but PBKDF2 hashes are not supported (yet).

### Describe what you'd like Logto to have

PBKDF2 user import support

**Comment by wangsijie** [comment]
This can be achieved by using `Legacy` algorithm, check this to learn the usage: https://github.com/logto-io/docs/pull/1099

**Comment by wangsijie** [comment]
Yes, you are right. This can not be implemented by `crypto.createHash`. So this is a valid feature request.

**Comment by wangsijie** [comment]
https://github.com/logto-io/logto/pull/7585

**Comment by PashaPash** [comment]
@wangsijie Is it possible to import users with non-ASCII salt bytes?
We’re migrating from ASP.NET Core Identity, where the salt is generated using rng.GetBytes(), so it contains arbitrary binary data.

**Comment by wangsijie** [comment]
Hi @PashaPash would you mind opening a new issue?
